### PR TITLE
ADD Developers Manual documentation

### PR DESCRIPTION
**Pull request by fgalan**
Twin of https://github.com/telefonicaid/fiware-orion/pull/2935 (for master)

Conflicts:
	doc/manuals/admin/metrics_api.md
	doc/manuals/admin/perf_tuning.md

### CODE REVIEW

File: doc/manuals/admin/perf_tuning.md [hunk 1]
```diff
@@ -8,7 +8,8 @@
 * [Thread pool considerations](#thread-pool-considerations)
 * [Identifying bootlenecks looking at semWait statistics](#identifying-bootlenecks-looking-at-semwait-statistics)
 * [Log impact in performance](#log-impact-in-performance)
-* [Mutex policy impact in performance](#mutex-policy-impact-in-performance)
+* [Metrics impact in performance](#metrics-impact-in-performance)
```
Comment (kzangeli) [inline doc/manuals/admin/perf_tuning.md]: This typo I guess we have in 'master' as well ...
impact in performance => impact on performance

Comment (fgalan) [inline doc/manuals/admin/perf_tuning.md]: The conflict wasn't correctly resolved. Fixed in https://github.com/telefonicaid/fiware-orion/pull/2940/commits/f2c9efb8a381e531ffc9152a8c65410da21ff03e

File: doc/manuals/admin/perf_tuning.md [hunk 2]
```diff
@@ -8,7 +8,7 @@
 * [Thread pool considerations](#thread-pool-considerations)
 * [Identifying bootlenecks looking at semWait statistics](#identifying-bootlenecks-looking-at-semwait-statistics)
```
Comment (kzangeli) [inline doc/manuals/admin/perf_tuning.md]: Here we have another typo:

bootleneck => bottleneck

File: doc/manuals/admin/perf_tuning.md [hunk 3]
```diff
@@ -8,7 +8,8 @@
...
 * [Log impact in performance](#log-impact-in-performance)
```
Comment (kzangeli) [inline doc/manuals/admin/perf_tuning.md]: Another "impact in"

File: doc/manuals/admin/perf_tuning.md (hunk 1 above)
Comment (kzangeli) [inline doc/manuals/admin/perf_tuning.md]: Typo fixed in 7a69b0a

File: doc/manuals/admin/perf_tuning.md (hunk 2 above)
Comment (kzangeli) [inline doc/manuals/admin/perf_tuning.md]: Typo fixed in 7a69b0a

File: doc/manuals/admin/perf_tuning.md (hunk 3 above)
Comment (kzangeli) [inline doc/manuals/admin/perf_tuning.md]: Typo fixed in 7a69b0a

**Comment by kzangeli** [comment]
LGTM

**Comment by fgalan** [comment]
This PR has got "dirty". Better to close and start again, see PR https://github.com/telefonicaid/fiware-orion/pull/2943

**Pull request closed without merging**